examples/storage/main.tf

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
+  credentials = "${file(var.credentials_path)}"
+}
+
+# Example of a org-level sink to a Cloud Storage bucket
+module "org_sink" {
+  source = "../../"
+  name   = "test-org-sink-gcs"
```

Comment:
It would be nice to be able to optionally specify a different prefix than `test-` for all of these different resources.

Reply:
@joestump This is just for the example, the module itself allows these properties to be configurable.